lib/active_merchant/billing/gateways/omise.rb

```diff
@@ -0,0 +1,298 @@
+require 'active_merchant/billing/rails'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class OmiseGateway < Gateway
+      API_VERSION = '1.0'
+      API_URL     = 'https://api.omise.co/'
+      VAULT_URL   = 'https://vault.omise.co/'
+
+      STANDARD_ERROR_CODE_MAPPING = {
```

Comment:
If your messages are consistent, I think parsing them is fine - it would be great to have more standard error codes coming out of this integration if we can.